pkg/cmd/ls.go

```diff
@@ -93,17 +93,11 @@ func NewLsCmd(targetReader TargetReader, configReader ConfigReader, ioStreams IO
 // printProjectsWithShoots lists list of projects with shoots
 func printProjectsWithShoots(target TargetInterface, ioStreams IOStreams) error {
```

Comment:
```go
func printProjectsWithShoots(target TargetInterface, ioStreams IOStreams) error
```
The signature indicates that an error is returned. This isn't the case anymore. 

Also I'm not sure if I like the fact that in case of error the process exits and the caller is not able to handle the error. Maybe the caller is not interested in the error in this case.. but I would not introduce this as default error handling pattern for gardenctl. Just something to think about..

Reply:
forced update by mistake. issue fixed